tests/onnxruntime/test_optimization.py

```diff
@@ -101,6 +103,52 @@ def test_optimization_details(self):
             gc.collect()
 
 
+class ORTQuantizerTest(unittest.TestCase):
+    LOAD_CONFIGURATION = {
+        "transformers_model": {
+            "model_name_or_path": "distilbert-base-uncased-finetuned-sst-2-english",
+            "from_transformers": True,
+            "task": "text-classification",
+        },
+        "optimum_model": {
+            "model_name_or_path": "optimum/distilbert-base-uncased-finetuned-sst-2-english",
+            "task": "text-classification",
+        },
+        "local_asset": {
+            "model_name_or_path": "tests/assets/onnx",
+            "task": "text-classification",
+        },
+        "ort_model_class": {
+            "model_name_or_path": ORTModelForSequenceClassification.from_pretrained(
+                "optimum/distilbert-base-uncased-finetuned-sst-2-english"
+            )
+        },
+    }
+
+    @parameterized.expand(LOAD_CONFIGURATION.items())
+    def test_from_pretrained_method(self, *args):
+        _, args = args
+        quantizer = ORTQuantizer.from_pretrained(**args)
+        self.assertIsInstance(quantizer, ORTQuantizer)
+
+    def test_fail_from_pretrained_method(self):
+        with self.assertRaises(Exception) as context:
+            ORTQuantizer.from_pretrained("bert-base-cased", from_transformers=True)
+
+        self.assertTrue("When using from_transformers, you need to provide a feature/task.", str(context.exception))
+        with self.assertRaises(Exception) as context:
+            ORTQuantizer.from_pretrained("optimum/distilbert-base-uncased-finetuned-sst-2-english")
+
+        self.assertTrue("Unable to load model", str(context.exception))
+
+        with self.assertRaises(Exception) as context:
+            ORTQuantizer.from_pretrained(
+                "optimum/distilbert-base-uncased-finetuned-sst-2-english", task="object-detection"
+            )
+
+        self.assertTrue("Unable to load model", str(context.exception))
```

Comment:
```suggestion
        self.assertIn("Unable to load model", str(context.exception))
```
The type of `assert` needs to be changed also.